packages/docs-site/src/pages/index.js

```diff
@@ -75,7 +75,8 @@ export default function Home() {
           <b>
             Penrose is an early-stage system that is actively in development.
           </b>{" "}
-          Feel free to <a href="mailto:team@penrose.ink">get in touch</a>.
+          Feel free to <a href="mailto:penrose-team@cs.cmu.edu">get in touch</a>
+          .
```

Comment:
Doesn't putting the `.` on the next line cause a space to appear in the rendered page?

Reply:
Prettier added that artistic touch for us, haha.  The good news is on modern browsers it renders fine.

Reply:
OK! As long as it works :)